modules/backend/lang/ar/lang.php

```diff
@@ -0,0 +1,588 @@
+<?php
+
+return [
+    'auth' => [
+        'title' => 'لوحة الإدارة'
+    ],
+    'field' => [
+        'invalid_type' => 'Invalid field type used :type.',
```

Comment:
Do not include the default translation strings in your PR. Only include those strings that you have already translated.